### PR TITLE
syncplay: Add version 1.6.9

### PR DESCRIPTION
**Pull request by Duckle29**
- Closes #2730

syncplay allows far away friends to sync their media players

### CODE REVIEW

File: bucket/syncplay.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.6.9",
+    "description": "Solution to synchronize video playback across multiple instances of mpv, VLC, MPC-HC, MPC-BE and mplayer2 over the Internet.",
+    "homepage": "https://syncplay.pl/",
+    "license": "Apache-2.0",
+    "url": "https://github.com/Syncplay/syncplay/releases/download/v1.6.9/Syncplay-Portable-Client-1.6.9.zip",
```
Comment (Ash258) [inline bucket/syncplay.json:6]: Why `portable`?

What does it mean? What needs to be persisted?

Comment (Duckle29) [reply bucket/syncplay.json:6]: Their page says the non-portable version stores connection settings in appdata

Comment (Duckle29) [reply bucket/syncplay.json:9]: Added persist for the client settings

**Pull request merged**